doc/source/development/contributing.rst

```diff
@@ -89,8 +89,8 @@ The issue will then show up to the pandas community and be open to comments/idea
 Working with the code
 =====================
 
-Now that you have an issue you want to fix, enhancement to add, or documentation to improve,
-you need to learn how to work with GitHub and the pandas code base.
+Now that you have an issue you want to fix, an enhancement to add, or documentation to improve,
+you need to learn how to work with GitHub and the pandas codebase.
```

Comment:
In computer science programming, we usually write Codebase and not Code base/ Code-Base. So I think this is not that necessary.